whd/pull_request.py

```diff
@@ -261,18 +263,40 @@ def set_pr_labels(
     }
     if not required_labels:
         return True
+
     pr_number = pr_event.number()
 
     repo = pr_event.repository()
     repository_path = repo.repository_path()
     owner, _ = repository_path.split('/')
+    job_mapping_set: model.concourse.JobMappingSet = cfg_set.job_mapping()
+    job_mapping = job_mapping_set.job_mapping_for_repo_url(
+        repo.repository_url(), cfg_set
+    )
+    trusted_teams = job_mapping.trusted_teams()
 
     sender_login = pr_event.sender()['login']
-    if pr_event.action() is PullRequestAction.OPENED:
-        if github_helper.is_pr_created_by_org_member(pr_number):
+    if pr_event.action() in PullRequestAction.OPENED:
```

Comment:
factor-out the checking-code into a separate function (check_if_label_is_required, or similarly-named)? this should (I think?) deduplicate (again) the new code introduced in this PR, should it not?